reporting/api/src/main/java/org/jboss/windup/reporting/model/ApplicationReportIndexModel.java

```diff
@@ -71,7 +71,8 @@
             List<ApplicationReportModel> reports = new ArrayList<>();
             for (ApplicationReportModel m : getApplicationReportModels())
             {
-                reports.add(m);
+                if (!m.getReportName().equals("Application List"))
```

Comment:
I added static link to index.html and I didn't get why Application List index report is there for global level reports so I filter result from getApplicationReportModelsSortedByPriority which is called in navBar.ftl (https://github.com/windup/windup/blob/master/reporting/impl/src/main/resources/reports/templates/include/navbar.ftl#L3).